etl/steps/data/grapher/regions/latest/regions.py

```diff
@@ -0,0 +1,305 @@
+"""Regions dataset adjusted for Grapher. It is published as CSV so that it can
+be easily loaded in Grapher codebase.
+
+This dataset is not meant to be imported to MySQL and is excluded from automatic deployment.
+"""
+
+import re
+
+from owid.catalog import Dataset, Table
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+# Legacy slugs for countries that have changed their name.
+LEGACY_SLUGS = {
+    "CZE": "czech-republic",
+    "MKD": "macedonia",
+    "SWZ": "swaziland",
+    "TLS": "timor",
+}
```

Comment:
So, are you using `LEGACY_SLUGS`? If not, you could remove it. And if it's needed, maybe it could be added as some new field, like `legacy_redirects` or something like that.

Reply:
Ouch, you're right. I removed them by accident. If redirects are doable then it would certainly be cleaner (though if we don't know or would be too complicated, it doesn't need to block this PR). Maybe we could just move legacy slugs to grapher so that it's closer to redirects once we decide to refactor it.

Reply:
Agreed. I'll make the slug substitutions on my end until it's time to set up redirects on wordpress.

Reply:
All right! Removed legacy slugs, ball is on your side Christian.